types.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module '*.css' {
```

Comment:
@JulianCataldo Have you tried this syntax instead? which is the PR I proposed.
You can't directly return a type